dist/index.html

```diff
@@ -5,7 +5,138 @@
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <main class="all-recipes">
+      <h1> Our recipes </h1>
+      <article class="recipe-grid">
```

Comment:
Should the main article we currently have here actually be a section?

Reply:
Done. It was actually also missing a closing tag so thank you for pointing that out.